tests/test_tools.py

```diff
@@ -2009,3 +2010,155 @@ def test_remove_duplicate_docs(inp, expected):
 def test_filter_employment_for_advisees(inp, expected):
     actual = filter_employment_for_advisees(inp[0], inp[1], "ms", dt.date(2021,6,3))
     assert actual == expected
+
+person1 = {"_id":"tstark", "aka":"iron man", "name":"tony stark"}
+person2 = {"_id":"nromanov", "aka":"black widow", "name":"natasha romanov"}
+PEOPLE = [person1, person2]
+
+presentation1 = {"_id":"abc", "authors":["tstark"], "begin_year":2018, "begin_month":1, "begin_day":1,
+                 "department":"apam", "institution":"columbiau", "status":"accepted", "type":"award"}
+presentation2 = {"_id":"ghi", "authors":["tstark","nromanov"], "begin_year":2019, "begin_month":1, "begin_day":2,
```

Comment:
please can you enter as dates here `begin_date`, `end_date` to test that pattern